draft-packages/modal/KaizenDraft/Modal/Primitives/ModalAccessibleLabel.scss

```diff
@@ -1,7 +1,27 @@
 @import "~@kaizen/design-tokens/sass/color";
 @import "~@kaizen/design-tokens/sass/border";
-.label {
+@import "~@kaizen/component-library/styles/responsive";
+
+$ca-breakpoint-small-mobile: 375px;
+
+@mixin ca-media-small-mobile {
+  @media (max-width: #{$ca-breakpoint-small-mobile - 1px}) {
+    @content;
+  }
+}
+
+.modalLabel {
   position: relative;
+  h2 {
+    @include ca-media-mobile {
+      font-size: 1.625rem;
+      line-height: 32px;
+    }
+    @include ca-media-small-mobile {
+      font-size: 1.5rem;
+      line-height: 30px;
+    }
```

Comment:
These seem very specific. Are they from Figma? Could we potentially use any other already in use vars?

Reply:
These were for styling the Headings responsively based on the Figma styles. Eventually they will be removed and replaced with the inherit responsive styles once that separate component is built out. 

I'll update to use vars if there are ones.